modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/view_asset_entries_table.jsp

```diff
@@ -21,198 +21,186 @@ if (stageableGroup.isLayout()) {
 }
 %>
 
-<clay:sheet>
-	<c:if test="<%= Validator.isNotNull(assetEntryResult.getTitle()) %>">
-		<clay:sheet-header>
-			<p class="h4 sheet-title">
-				<%= assetEntryResult.getTitle() %>
-			</p>
```

Comment:
Hey @matuzalemsteles we shouldn't be removing the title. We can just add this above the table with just the paragraph.
```
<c:if test="<%= Validator.isNotNull(assetEntryResult.getTitle()) %>">
	<p class="h4">
		<%= assetEntryResult.getTitle() %>
	</p>
</c:if>
```

Reply:
@ethib137 Maybe the design of this could be a little strange. Because we still have the portlet title so I don't know how this can look visually with two titles together...

![Screenshot 2024-02-06 at 08 58 31](https://github.com/liferay-platform-experience/liferay-portal/assets/13750819/bfd3a5e7-ac9f-40f3-9a65-4eab578e7ea4)

I tried to see if there is any configuration for this portlet to try to render the title but without success. Do you have any idea about this?

Reply:
Hey @matuzalemsteles ... I think it's okay... the "Asset Publisher" title is only rendered on widget pages, not on content pages. So, we won't normally have 2 titles side by side. Additionally, I also couldn't find a way to get `assetEntryResult.getTitle()` to be displayed. I searched the code and couldn't find anywhere that even sets the title, so I think it's likely no longer ever rendered. But, all the other display templates for asset publisher still include it, so our small refactor of the table shouldn't remove it. Does that sound okay?